### PR TITLE
Release new 2.0.0 RC

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by S11001001 — APPROVED** [review]

File: LATEST [hunk 1]
```diff
@@ -1,6 +1,6 @@
 d96b7bdc93379fd60f7292bf9e0b50e9573a566a 2.1.0-snapshot.20220223.9363.0.d96b7bdc SPLIT_RELEASE
 4038d0a7e3694305ba477cd9a4268400c39c20cd 2.0.0-snapshot.20220127.9042.0.4038d0a7
-b7fc9f5731aae9fdde364e6745dfb47ee1e10030 2.0.0-snapshot.20220209.9212.0.b7fc9f57 SPLIT_RELEASE
+b291a57e5c1c8fbed19dc1ffc3a7bce1bd121e04 2.0.0-snapshot.20220225.9368.0.b291a57e
```
Comment (garyverhaegen-da) [inline LATEST:3]: It still needs to be a SPLIT_RELEASE.

**Review by garyverhaegen-da — CHANGES_REQUESTED** [review]
We need a split release & auto-merge is enabled.

File: LATEST (hunk 1 above)
Comment (cocreature) [reply LATEST]: fixed

**Review by garyverhaegen-da — APPROVED** [review]

File: LATEST (hunk 1 above)
Comment (cocreature) [reply LATEST]: clearly shouldn’t be working at this point

Comment (cocreature) [reply LATEST]: Maybe also worth adding a check to disallow non-split releases for newer versions.

**Pull request merged**